src/main/java/ch/heigvd/dai/Main.java

```diff
@@ -1,7 +1,32 @@
 package ch.heigvd.dai;
 
+import ch.heigvd.dai.commands.Root;
+import java.io.File;
+import picocli.CommandLine;
+
 public class Main {
   public static void main(String[] args) {
-    System.out.println("Hello world!");
+    // Define command name - source: https://stackoverflow.com/a/11159435
+    String jarFilename =
+        new File(Main.class.getProtectionDomain().getCodeSource().getLocation().getPath())
```

Comment:
I believe some of the examples we saw in class also make use of this line. You could maybe reference them too

Reply:
Actually these lines are simply copy-paste from the class example! 😅

Reply:
No wonder 🤣